plugins/modules/zos_operator.py

```diff
@@ -142,23 +181,140 @@ def parse_params(params):
         cmd=dict(arg_type="str", required=True),
         verbose=dict(arg_type="bool", required=False),
         debug=dict(arg_type="bool", required=False),
+        security=dict(arg_type="bool", required=False),
+        delay=dict(arg_type="int", required=False),
+        reset=dict(arg_type="int", required=False),
     )
     parser = BetterArgParser(arg_defs)
     new_params = parser.parse_args(params)
     return new_params
 
 
 def run_operator_command(params):
+    script = """/*rexx*/
+Address 'TSO'
+IsfRC = isfcalls( "ON" )
+if __argv.0 < 4 then
```

Comment:
I originally wrote this as a stand-alone script so I could learn the language, test quickly, and then pasted it into the python code.  "locode" was when I was looking for why the usage routine was called.  It's valid that the usage information should never fire, and won't go anywhere now, so I'll pull that out.  I'll look up function arguments... I defaulted to argv[] because it was stand-alone code... I'll check for better examples.

Reply:
I've rewritten the rexx to use args for the first few (fixed position) values.. the - options can be in any order, after the command.

Reply:
@richp405 I figure and I would have done the same, written as a stand alone to learn and debug, good way to handle this modification ant it looked like it was standalone that just needed a bit of attention :) 👍

Reply:
I was trying to make the flags non-position-centric.  Let me know about the -d/debug trace option, and I'll make the changes today.

Reply:
Yup, we talked and opted to remove the `-d` , not even that particular on the function , was just commenting it as a no-op.